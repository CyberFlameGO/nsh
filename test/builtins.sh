source
source test/builtins.sh.source
echo $msg

func1() {
    echo $1
    shift
    echo $1
    shift
    echo $1
    shift
    echo $1
    shift
    echo $1
    shift
    exit 0
}

func1 a b "c d"
