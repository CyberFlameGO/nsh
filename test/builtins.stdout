source command works
a
b
c d
